template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-FSharp/Company.WebApplication1.fsproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' == ''">netcoreapp2.0</TargetFramework>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Controllers/*.fs" />
```

Comment:
Hmm, this is a bit silly, but I'm unsure of what the proper thing is here. Due to the significance of compilation order in F#, it usually works out better if one explicitly lists the files in the order that they need to be compiled unless one is certain that there are no dependencies between the files. However, controllers typically do not have a dependencies on one another, so...dunno. Put simply, since wildcard file inclusion was not possible before, I don't think anyone has had to answer the question of whether it is reasonable to rely on it if one has good reason not to expect dependencies between files...